inc/commonitilobject.class.php

```diff
@@ -2717,142 +2717,211 @@ static function processMassiveActionsForOneItemtype(MassiveAction $ma, CommonDBT
    function getSearchOptionsMain() {
       global $CFG_GLPI;
 
-      $tab                            = array();
-      $tab['common']                  = __('Characteristics');
-
-      $tab[1]['table']                = $this->getTable();
-      $tab[1]['field']                = 'name';
-      $tab[1]['name']                 = __('Title');
-      $tab[1]['datatype']             = 'itemlink';
-      $tab[1]['searchtype']           = 'contains';
-      $tab[1]['massiveaction']        = false;
-      $tab[1]['additionalfields']     = array('id', 'content', 'status');
-
-      $tab[21]['table']               = $this->getTable();
-      $tab[21]['field']               = 'content';
-      $tab[21]['name']                = __('Description');
-      $tab[21]['massiveaction']       = false;
-      $tab[21]['datatype']            = 'text';
+      $tab = [];
```

Comment:
Yes, but I'm not sure we really want to keep those ones (will require much more work).